bin/common.c

```diff
@@ -28,6 +28,7 @@
 
 #include "api/s2n.h"
 #include "error/s2n_errno.h"
+#include "stuffer/s2n_stuffer.h"
```

Comment:
We don't use stuffers in the examples because they're only supposed to use public methods. We probably violate this in a few places, but we shouldn't add more violations.

Reply:
At least I got some practice in working with stuffers :-)